src/main/java/com/truelayer/java/entities/accountidentifier/AccountIdentifier.java

```diff
@@ -58,52 +58,52 @@
     }
 
     @JsonIgnore
-    public boolean isSortCodeAccountNumber() {
+    public boolean isSortCodeAccountNumberIdentifier() {
         return this instanceof SortCodeAccountNumberAccountIdentifier;
     }
 
     @JsonIgnore
-    public boolean isIban() {
```

Comment:
For this and below ones: a `@JsonIgnore` on a `isX` masks the value of `X` in the specific type 😱